src/app/cluster/details/cluster/edit-cluster/template.html

```diff
@@ -104,7 +104,8 @@
       </div>
 
       <mat-checkbox *ngIf="isKonnectivityEnabled"
-                    [formControlName]="Controls.Konnectivity">
+                    [formControlName]="Controls.Konnectivity"
+                    [class.km-text-muted]="form.get(Controls.Konnectivity).disabled">
```

Comment:
I just noticed we are not using `[class.<class-name>]` syntax anywhere better to use `ngClass`

Reply:
I think its better to use `ngClass` as Khizer suggested since we don't use other syntax anywhere else